test/mlir/krnl/krnl_normalization.mlir

```diff
@@ -14,3 +14,17 @@ func @test_krnl_memcpy_norm(%arg0: memref<1x16384xf32>) -> memref<1x16x4x4xf32,
   return %0 : memref<1x16x4x4xf32, #map_tile>
   // CHECK: return [[ALLOC]] : memref<1x16x1x1x32x32xf32>  
 }
+
+// CHECK-LABEL: test_getref_norm
+func @test_getref_norm() ->  () {
+  %c0_i64 = constant 0 : i64
+  %0 = alloc() : memref<1x81920xf32>
+  %1 = alloc() : memref<1x16x4x4xf32, #map_tile>
+  // CHECK: [[ALLOC:%.+]] = alloc() : memref<1x16x1x1x32x32xf32>
+  %2 = "krnl.getref"(%0, %c0_i64) : (memref<1x81920xf32>, i64) -> memref<1x16x4x4xf32>
```

Comment:
How does this test normalization trait for getref?

Reply:
@doru1004  All arguments and results of `getref` don't have map in the memrefs, but in current MLIR implementation, if there is an non-normalizable operation (in this case `getref`) in the func op, all of map in the func op are not normalized by `--normalize-memrefs`.  So, `MemRefsNormalizable` need to be added in the dialect to have it normalizable operation.

- This is normalized (comment out `getref`), but if `getref` is inserted, this is not normalized. 
```
func @test_getref_norm() ->  () {
  %c0_i64 = constant 0 : i64
  %0 = alloc() : memref<1x81920xf32>
  %1 = alloc() : memref<1x16x4x4xf32, #map_tile>
  // %2 = "krnl.getref"(%0, %c0_i64) : (memref<1x81920xf32>, i64) -> memref<1x16x4x4xf32>
  dealloc %1: memref<1x16x4x4xf32, #map_tile>
  dealloc %0: memref<1x81920xf32>
  return
}
```

Reply:
Got it! Thanks for the explanation! This makes sense now!